include/nffs/os.h

```diff
@@ -21,9 +21,29 @@
 #define H_OS_
 
 #include <stdint.h>
-#if __ZEPHYR__
+
+#if MYNEWT
+
+#include "fs/fs.h"
+#include "fs/fs_if.h"
+struct fs_ops nffs_ops;
```

Comment:
I would avoid non-static variable definition in common header. Doesn't it generate warning on unused variable in few files here?

Reply:
Yeah, this is a bit weird, but it is what's called a "tentative definition", so it should work as long as there is a real definition in some translation unit (which is the case). Let's say I made this a real definition, by attributing a value as in `struct fs_ops nffs_ops = { 0 };`, this would actually fail the build. I guess I will just add `extern` to the line above with the hopes of making it clearer!

Reply:
You can check the CI: https://travis-ci.org/apache/mynewt-nffs/jobs/469953470

Reply:
Btw, that variable is just a table of functions calls. One copy or 100 won't make much difference as long as the functions are correctly pointed.